File: app/src/main/java/me/devsaki/hentoid/json/sources/EHentaiImageResponse.java
package me.devsaki.hentoid.json.sources;

@SuppressWarnings({"unused, MismatchedQueryAndUpdateOfCollection", "squid:S1172", "squid:S1068"})
public class EHentaiImageResponse {
    private String lf;
    private String ls;
    private String ll;
    private String lo;
    private String i; // image displayed in the multipage viewer
    private String s;

    public String getUrl() {
        return i;
    }
}

File: app/src/main/java/me/devsaki/hentoid/json/sources/ehentai/EHentaiImageResponse.kt
package me.devsaki.hentoid.json.sources.ehentai

import com.squareup.moshi.Json

data class EHentaiImageResponse(
    // Relative link to the full-size image
    val fullUrlRelative: String? = null,
    val ls: String? = null,
    val ll: String? = null,
    val lo: String? = null,
    // image displayed in the multipage viewer
    @Json(name = "i")
    val url: String,
    val s: String? = null
)